src/components/Badge/Badge.tsx

```diff
@@ -20,7 +20,7 @@ export interface Props {
    * - **teal** results in a badge with teal state
    * - **dark-gray** results in a sparkline text bubble with normal results
    */
-  variant: 'warning' | 'error' | 'success';
+  variant?: 'warning' | 'error' | 'success';
```

Comment:
I changed this because of a TS warning I saw when I opened this file; it doesn't relate to the import work in this PR and can be moved to a separate PR if requested.